core/src/main/java/bio/terra/pearl/core/dao/dataimport/ImportDao.java

```diff
@@ -18,4 +22,9 @@ protected Class<Import> getClazz() {
         return Import.class;
     }
 
+    public List<Import> findAllByStudyEnv(UUID studyEnvId) {
+        return super.findAllByProperty("study_environment_id", studyEnvId);
```

Comment:
the 'super' is redundant
```suggestion
        return findAllByProperty("study_environment_id", studyEnvId);
```